scripts/tokenomics-emulation/emulations/cere-app-to-user-emulation.ts

```diff
@@ -0,0 +1,29 @@
+import { IEmulation } from "./emulation.interface";
+import Accounts from "../accounts";
+import CereSmartContract from "../cere-smart-contract";
+
+class CereApptoUserEmulation implements IEmulation {
+  constructor(
+    private readonly config,
+    private readonly account: Accounts,
```

Comment:
You should pass only account from which you are going to send tokens.

Reply:
I have passed the whole class because we are using `generateSrAccounts` to generate a new account.